crates/sgx-attestation/src/dcap/quote.rs

```diff
@@ -0,0 +1,369 @@
+use alloc::borrow::ToOwned;
```

Comment:
There is a lot of duplication in this file with `get_collateral/parse_quote.rs`. Might be better to merge `parse_quote.rs` into this one. However, be careful there is not much security consideration in `parse_quote.rs` since it isn't the critical part.